### PR TITLE
python3Packages.adb-shell: 0.3.0 -> 0.3.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.3.1

- Switched to `pytestCheckHook`
- Ran `nixpkgs-fmt`
- Added `pythonImportsCheck`

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113393` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.adb-shell</li>
<li>python37Packages.androidtv</li>
<li>python38Packages.adb-shell</li>
<li>python38Packages.androidtv</li>
<li>python39Packages.adb-shell</li>
<li>python39Packages.androidtv</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

python37Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python37Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>
python38Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python38Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>
python39Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python39Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113393` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.adb-shell</li>
<li>python37Packages.androidtv</li>
<li>python38Packages.adb-shell</li>
<li>python38Packages.androidtv</li>
<li>python39Packages.adb-shell</li>
<li>python39Packages.androidtv</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

python37Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python37Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>
python38Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python38Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>
python39Packages.adb-shell:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:

```
   |
12 | , python
   |   ^
```
</details>
python39Packages.androidtv:
<details>


Unused argument: `python`.
Near pkgs/development/python-modules/androidtv/default.nix:10:3:

```
   |
10 | , python
   |   ^
```
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113393` at 390ab80e run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/bab249aa0eb83d8a0863a1d2efc61ddf">python37Packages.adb-shell</a></li></li>
    <li><a href="https://gist.github.com/0a7026843b5e61e4cdffa1cbe2240d5a">python37Packages.androidtv</a></li></li>
    <li><a href="https://gist.github.com/e1f13d03a675f4ff94475a0d66b7be14">python38Packages.adb-shell</a></li></li>
    <li><a href="https://gist.github.com/6f18523c858ca31f8b64dbf2adbe05df">python38Packages.androidtv</a></li></li>
    <li><a href="https://gist.github.com/99f2d42c5be5cdced40391ec377b0b88">python39Packages.adb-shell</a></li></li>
    <li><a href="https://gist.github.com/14469d819918c2aa05743c9d6eae40d5">python39Packages.androidtv</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `python`.
Near pkgs/development/python-modules/adb-shell/default.nix:12:3:
```
   |
12 | , python
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/adb-shell/default.nix [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+, buildPythonPackage
+, cryptography
+, fetchFromGitHub
+, isPy3k
+, libusb1
+, mock
+, pyasn1
+, pycryptodome
+, pytestCheckHook
+, python
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/adb-shell/default.nix:12]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/adb-shell/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/adb-shell/default.nix]: Removed, thanks.

**Pull request merged**